Cargo.toml

```diff
@@ -31,13 +31,12 @@ dusk-jubjub = "0.8"
 itertools = "0.9.0"
 rand_chacha = "0.2"
 rayon = "1.3.0"
-anyhow = "1.0.32"
-thiserror = "1.0"
 serde = "1.0"
 # Dusk related deps for WASMI serde
 canonical = {version = "0.5", optional = true}
 
 [dev-dependencies]
+anyhow = "1.0.32"
```

Comment:
This should not be required. Why do we need that?

Reply:
Mostly for ease of testing - with the centralized error file you suggested, I don't see this as necessary though, so I will remove it.